rohmu/__init__.py

```diff
@@ -6,13 +6,15 @@
 """
 from .errors import InvalidConfigurationError
 from .object_storage.base import BaseTransfer
-from typing import Type
+from typing import Any, Dict, Type
 
 IO_BLOCK_SIZE = 2**20  # 1 MiB
+STORAGE_TYPE = "storage_type"
+Config = Dict[str, Any]
```

Comment:
This should take a `Mapping` instead of a `Dict`, to make it clear that the input parameter will not be modified.